===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/main/glheader.h,v
rcsdiff: /cvs/mesa/Mesa/src/mesa/main/glheader.h,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.70
retrieving revision 1.71
diff -u -r1.70 -r1.71
--- ../Mesa-6.4.2/src/mesa/main/glheader.h	2006/02/11 02:59:38	1.70
+++ ../Mesa-6.4.2/src/mesa/main/glheader.h	2006/02/15 21:03:34	1.71
@@ -46,6 +46,9 @@
 #ifndef GLHEADER_H
 #define GLHEADER_H
 
+#ifdef HAVE_DIX_CONFIG_H
+#include "dix-config.h"
+#endif
 
 #if defined(XFree86LOADER) && defined(IN_MODULE) && !defined(NO_LIBCWRAPPER)
 #include "xf86_ansic.h"
